code/web/src/modules/common/header/Header.js

```diff
@@ -1,90 +1,106 @@
 // Imports
-import React from 'react'
-import PropTypes from 'prop-types'
-import { connect } from 'react-redux'
-import { withRouter } from 'react-router-dom'
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
 
 // UI Imports
-import { Grid, GridCell } from '../../../ui/grid'
-import { primary as primaryGradient } from '../../../ui/common/gradients'
-import { level1 } from '../../../ui/common/shadows'
+import { Grid, GridCell } from '../../../ui/grid';
+import { primary as primaryGradient } from '../../../ui/common/gradients';
+import { level1 } from '../../../ui/common/shadows';
 
 // App Imports
-import home from '../../../setup/routes/home'
-import user from '../../../setup/routes/user'
-import crate from '../../../setup/routes/crate'
-import admin from '../../../setup/routes/admin'
-import Logo from './Logo'
-import Menu from './Menu'
-import MenuItem from './MenuItem'
+import home from '../../../setup/routes/home';
+import user from '../../../setup/routes/user';
+import crate from '../../../setup/routes/crate';
+import admin from '../../../setup/routes/admin';
+import Logo from './Logo';
+import Menu from './Menu';
+import MenuItem from './MenuItem';
+import Button from '../../../ui/button/Button';
+import { logout } from '../../user/api/actions';
 
 // Component
 const Header = (props) => {
   return (
-    <header style={{
-      backgroundImage: primaryGradient,
-      boxShadow: level1,
-      padding: '0 2em',
-      height: '5em',
-      position: 'fixed',
-      left: 0,
-      right: 0,
-      top: 0
-    }}>
+    <header
+      style={{
```

Comment:
Seems to be a lot of changes in this file, is this just spacing?

Reply:
It's spacing. Only real change I see os a logout button